test.md

```diff
@@ -0,0 +1 @@
+"This file should be ignored by git"
```

Comment:
- [ ] Make sure that this file is ignored by git and is not present in your GitHub repository in order to apply the project requirements.